src/bootstrap.js

```diff
@@ -23,7 +23,12 @@ module.exports = (ceConfig = {}, pnConfig = {}) => {
 
     ChatEngine.pnConfig.heartbeatInterval = ChatEngine.pnConfig.heartbeatInterval || 30;
     ChatEngine.pnConfig.presenceTimeout = ChatEngine.pnConfig.presenceTimeout || 60;
-    // ChatEngine.pnConfig.ssl = true;
+
```

Comment:
If possible it would be best to leave SSL on by default for folks and turn it off in the test cases. But not a huge blocker for now to get tests passing!!